edi_account/tests/test_edi.py

```diff
@@ -0,0 +1,108 @@
+from odoo.addons.component.tests.common import SavepointComponentRegistryCase
+
+_logger = logging.getLogger(__name__)
+
+
+class EDIBackendTestCase(SavepointComponentRegistryCase, AccountTestInvoicingCommon):
+    @classmethod
+    def setUpClass(cls, chart_template_ref=None):
+        super().setUpClass(chart_template_ref=chart_template_ref)
+
+        tax_repartition_line = cls.company_data[
+            "default_tax_sale"
+        ].refund_repartition_line_ids.filtered(
+            lambda line: line.repartition_type == "tax"
+        )
+        cls.test_move = cls.env["account.move"].create(
+            {
+                "type": "entry",
+                "date": fields.Date.from_string("2016-01-01"),
+                "line_ids": [
+                    (
+                        0,
+                        None,
+                        {
+                            "name": "revenue line 1",
+                            "account_id": cls.company_data[
+                                "default_account_revenue"
+                            ].id,
+                            "debit": 500.0,
+                            "credit": 0.0,
+                        },
+                    ),
+                    (
+                        0,
+                        None,
+                        {
+                            "name": "revenue line 2",
+                            "account_id": cls.company_data[
+                                "default_account_revenue"
+                            ].id,
+                            "debit": 1000.0,
+                            "credit": 0.0,
+                            "tax_ids": [
+                                (6, 0, cls.company_data["default_tax_sale"].ids)
+                            ],
+                        },
+                    ),
+                    (
+                        0,
+                        None,
+                        {
+                            "name": "tax line",
+                            "account_id": cls.company_data[
+                                "default_account_tax_sale"
+                            ].id,
+                            "debit": 150.0,
+                            "credit": 0.0,
+                            "tax_repartition_line_id": tax_repartition_line.id,
+                        },
+                    ),
+                    (
+                        0,
+                        None,
+                        {
+                            "name": "counterpart line",
+                            "account_id": cls.company_data[
+                                "default_account_expense"
+                            ].id,
+                            "debit": 0.0,
+                            "credit": 1650.0,
+                        },
+                    ),
+                ],
+            }
+        )
+
+    def setUp(self):
+        super().setUp()
+        self._load_module_components("component_event")
```

Comment:
you can do this in setUpClass. BTW could you review this? https://github.com/OCA/connector/pull/387

Reply:
I tried to do it on setUpClass and an error was raised :cry: I can try to find why

Reply:
https://github.com/OCA/edi/pull/264/files#diff-1917008f7c39079edbeb6a81a96e98304bff5e2d327ea0443a960a3eed4567ccR90 :wink: